packages/core/src/App/Components/Layout/Footer/toggle-settings.jsx

```diff
@@ -23,20 +24,25 @@ const LanguageSettingContainer = Loadable({
 });
 
 const ModalContent = ({ settings_extension }) => {
-    const content = [
-        {
-            icon: 'IcTheme',
-            label: localize('Themes'),
-            // eslint-disable-next-line react/display-name
-            value: ThemeSetting,
-        },
-        {
-            icon: 'IcLanguage',
-            label: localize('Language'),
-            value: LanguageSettingContainer,
-        },
-        ...(settings_extension || []),
-    ];
+    const { is_pre_appstore } = React.useContext(PlatformContext);
+    const content = [];
+    if (is_pre_appstore) {
+        content.push(...(settings_extension || []));
+    } else {
+        content.push(
+            {
+                icon: 'IcTheme',
+                label: localize('Themes'),
+                // eslint-disable-next-line react/display-name
```

Comment:
[Question]: Why do we need this line?

Reply:
This for the default themes and language settings we have in deriv app.